typeql/language/define.feature

```diff
@@ -1617,8 +1617,48 @@ Feature: TypeQL Define Query
       };
       """
 
-
-  Scenario: redefining an existing rule updates its definition
+  Scenario: redefining a rule and querying updates its definition
+    Given typeql define
+      """
+      define
+      nickname sub attribute, value string;
+      person owns nickname;
+      rule people-bob:
+      when {
+        $p isa person;
+      } then {
+        $p has nickname "Bob";
+      };
+      """
+    Then transaction commits
+    When session opens transaction of type: write
+    Given typeql define
+      """
+      define
+      nickname sub attribute, value string;
+      person owns nickname;
+      rule people-bob:
+      when {
+        $p isa person, has email "bob@gmail.com";
```

Comment:
Maybe we should change this, so that the assertion later on is truly verifying that the rule definition was actually updated.
```suggestion
        $p has email "bob@gmail.com";
```